.pre-commit-config.yaml

```diff
@@ -18,7 +18,7 @@ repos:
     pass_filenames: false
   - id: typecheck
     name: Typecheck
-    entry: make typecheck-fast
-    types: [python]
-    language: system
+    entry: pyright pydantic
```

Comment:
surely we should just have one `make typecheck` which switches to use `pyright`?

Reply:
okay, I see you've done that. I would prefer to stick with `language: system` and running the `make` command.

Reply:
pyright is a node package. if we do that we'd have to somehow manage the node package ourselves. if we are already using pre-commit which can handle node packages, why not use that?

Reply:
Sorry, I was wrong: that is what I was intending to do but had not done it.